server/balancer.go

```diff
@@ -370,3 +372,262 @@ func (r *replicaChecker) checkBestReplacement(region *RegionInfo) Operator {
 	}
 	return newTransferPeer(region, oldPeer, newPeer)
 }
+
+// RegionStat records each hot region's statistics
+type RegionStat struct {
+	RegionID       uint64    `json:"region_id"`
+	WrittenBytes   uint64    `json:"written_bytes"`
+	HotDegree      int       `json:"hot_degree"`
+	LastUpdateTime time.Time `json:"last_update_time"`
+	StoreID        uint64    `json:"-"`
+	antiCount      int
+	version        uint64
+}
```

Comment:
Yep. I know what they do generally, but I think there should be some explanation here, for potential future readers.